### PR TITLE
Added Hyperlink to "Devtools" text

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
Added a hyperlink for the "DevTools" keyword.
Link - https://flutter.dev/docs/development/tools/devtools/

**Before Changes**
before changes

**After Changes**
After changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Dart & Flutter DevTools
+# Dart & Flutter [DevTools](https://flutter.dev/docs/development/tools/devtools/)
```
Comment (kenzieschmoll) [inline README.md:1]: Let's remove this hyperlink

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 [![Build Status](https://github.com/flutter/devtools/workflows/devtools/badge.svg)](https://github.com/flutter/devtools/actions)
 
 ## What is this?
 
-Dart DevTools is a suite of performance tools for Dart and Flutter.
+Dart [DevTools](https://flutter.dev/docs/development/tools/devtools/) is a suite of performance tools for Dart and Flutter.
```
Comment (kenzieschmoll) [inline README.md:7]: and replace `Dart [DevTools]` with `[Dart & Flutter DevTools]` here

File: README.md (hunk 1 above)
Comment (Pradumnasaraf) [reply README.md]: Done

**Comment by Pradumnasaraf** [comment]
@kenzieschmoll Please check

**Review by kenzieschmoll — APPROVED** [review]

**Comment by Pradumnasaraf** [comment]
Thank You.

**Pull request merged**